GenderPayGap.Core/Global.cs

```diff
@@ -131,10 +131,10 @@ public static bool SkipSpamProtection
                 {"X-Content-Security-Policy", "$(Content-Security-Policy)"},
                 {"Referrer-Policy", "origin-when-cross-origin"},
                 {"Strict-Transport-Security", "max-age=31536000; includeSubDomains"},
-                {"X-Powered-By", null},
-                {"X-AspNet-Version", null},
-                {"X-AspNetMvc-Version", null},
-                {"Server", null}
+                {"X-Powered-By", ""},
+                {"X-AspNet-Version", ""},
+                {"X-AspNetMvc-Version", ""},
+                {"Server", ""}

```

Comment:
Did this work before the change? What did the bug look like?

Reply:
This was originally in config (and worked).
Then, I moved it into code, with null values (which didn't work).
The error was one of those Asp.Net errors saying that some middleware threw an exception